lua/graph.lua

```diff
@@ -1618,21 +1622,18 @@ function filter_tags_generic(kv)
     kv["seasonal"] = "true"
   end
 
+  kv["hov_tag"] = "true"
```

Comment:
can you explain what is going on here. it reads like we are saying that all ways are always tagged wrt hov (ie we dont have to infer hov-ness the tag always is there to specify it).

Reply:
Ah yes.  I remain puzzled myself the true intention behind all the `auto_tag`, `hov_tag`, etc.  I mean, throughout the lua we check them for `nil` and we set them to `true/false`... but weirdly, the `tag_handler` logic doesn't care what you set the `hov_tag` value to, only that it is set:

```
      tag_handlers_["hov_tag"] = [this]() {
      osm_access_.set_hov_tag(true);
      has_user_tags_ = true;
    };
```
Once set, there is later logic (can't recall where) that says "hey, the access bit is set for hov, I need to query the way for the `hov_forward/hov_reverse` flags".

So I figured `hov_tag` can always be set - and what is most important is that the `hov_forward/hov_reverse` flags are correct. 

But it's quite possible I'm missing the wisdom/intention behind these `*_tag`'s.